Userland/Libraries/LibGUI/VimEditingEngine.cpp

```diff
@@ -63,8 +63,112 @@ bool VimEditingEngine::on_key_in_insert_mode(const KeyEvent& event)
     return false;
 }
 
+char VimEditingEngine::numeric_key_value(const KeyCode key)
```

Comment:
Instead of this, you can get the numeric character from the `KeyEvent::code_point()`

Reply:
Ah, that's an older commit, sorry!

I changed it to use a new method called "is_multiplied_key" in VimEditingEngine.cpp to apply it on the correct commands.
And "key_code_to_numeric_key_value" (returns Optional<T>) in VimEditingEngine.h (not sure if this one already exists, don't think I found it in the KeyEvent source?) to get the numeric value of the key code (if any)

If I'm understanding you correctly about the squashing, the first 3 commits should have been squashed into a first commit and the last 2 should have been squashed into a second commit? I'll do better next time, thanks for the advice :)